toolchain/gcc/gcc.cmake

```diff
@@ -37,6 +37,7 @@ ExternalProject_Add_Step(gcc final
         rustup
     COMMAND ${MAKE}
     COMMAND ${MAKE} install-strip
+    COMMAND bash -c "rm -rf ${MINGW_INSTALL_PREFIX}/bin/{ld,ld.bfd}"
```

Comment:
This not needed..compiler always start with cross-* prefix

Reply:
There's no ld or ld.bfd file in that bin folder, so that rm command is just pointless

Reply:
> currently the bfd wrapper is unused

How you check it?